satpy/readers/hrit_base.py

```diff
@@ -320,11 +321,12 @@ def read_band(self, key, info):
         elif self.mda['number_of_bits_per_pixel'] in [8, 10]:
             dtype = np.uint8
         shape = (shape, )
-        data = np.memmap(self.filename, mode='r',
-                         offset=self.mda['total_header_length'],
-                         dtype=dtype,
-                         shape=shape)
-        data = da.from_array(data, chunks=shape[0])
+        with utils.unzip_context(self.filename) as fn:
```

Comment:
Can you please add a comment here that `unzip_context` is more performant than `generic_open`?